adapters/rest-adapter-generator/src/test/java/uk/gov/justice/services/adapters/rest/generator/RestAdapterGenerator_POSTMethodBodyTest.java

```diff
@@ -172,18 +179,27 @@ public void shouldPassMapWithOnePathParamToRestProcessor() throws Exception {
 
     @SuppressWarnings({"unchecked", "rawtypes"})
     @Test
-    public void shouldPassMapWithOnePathParamToRestProcessorWhenInvoking2ndMethod() throws Exception {
+    public void shouldInvoke2ndMethodAndPassMapWithOnePathParamToRestProcessor() throws Exception {
 
         generator.run(
                 restRamlWithDefaults().with(
                         resource("/some/path/{p1}", "p1")
-                                .with(action(POST, "application/vnd.ctx.command.cmd-aa+json", "application/vnd.ctx.command.cmd-bb+json"))
+                                .with(httpAction(POST,
+                                        "application/vnd.cmd-aa+json",
+                                        "application/vnd.cmd-bb+json")
+                                        .with(mapping()
+                                                .withName("cmd-aa")
+                                                .withRequestType("application/vnd.cmd-aa+json"))
+                                        .with(mapping()
+                                                .withName("cmd-b")
```

Comment:
Is there a reason why this is cmd-b not cmd-bb ? It looks like this is intentionally different for the sake of the test but it is not clear why or what side-effects this will have.